examples/test/BasicComponents_test.js

```diff
@@ -94,3 +94,12 @@ test('Scatter plot should render', async (t) => {
         .expect(chart.exists).ok()
         .expect(chart.textContent);
 });
+
+test('Bubble chart should render', async (t) => {
+    const loading = Selector('.s-loading');
```

Comment:
what would you think of having a helper like (not tested):
```
async function checkRenderChart(selector) {
    const loading = Selector('.s-loading');
    const chart = Selector(selecor);
    await t
        .expect(loading.exists).ok()
        .expect(chart.exists).ok()
        .expect(chart.textContent);
}
```
and then just call from those tests above (just those which do exactly this)
```
test('Bubble chart should render', async (t) => {
    await checkRenderChart('.s-bubble-chart')
}
```